bin/functions

```diff
@@ -108,17 +108,44 @@ function install {
             case ${ASDF_INSTALL_VERSION} in
                 zulu*) mv ./* "${ASDF_INSTALL_PATH}" ;;
                 liberica*) mv ./* "${ASDF_INSTALL_PATH}" ;;
-                *) mv Contents/Home/* "${ASDF_INSTALL_PATH}" ;;
+                *) 
+                    mv Contents/Home/* "${ASDF_INSTALL_PATH}" 
+                    macOS_integration
```

Comment:
@halcyon Would it make sense to make this a configurable behavior?

Reply:
I think it's a good idea to make it configurable. 

Would it be opt in or opt out?

Reply:
My preference would be opt in. It would be good to preserve asdf's default behavior, and provide a configuration escape hatch for those who would prefer the location to fit in with the MacOS system default.

Reply:
Would the opt in be based on an environment variable? In this case, what name and value should I expect?

Reply:
Can we read it from [`$HOME/.asdfrc`](https://asdf-vm.com/#/core-configuration?id=homeasdfrc)?
There should be some functions for this already in asdf-vm.

How about `java_enable_macos_integration` for the setting name?

Reply:
I like it. I would suggest the slight adjustment `java_macos_integration_enable` to leave things open for the unlikely need for other java_macos_integration settings